.pack.mk

```diff
@@ -22,8 +22,8 @@ ifeq (${VERSION}, $(filter ${VERSION}, 1.10))
 endif
 
 prepare:
-#	rm -rf build packpack
```

Comment:
These changes shouldn't be introduced at all. 

You change this in the first commit and fix this here, which is wrong. Just don't change this in the first commit.

Reply:
done